contractcourt/channel_arbitrator.go

```diff
@@ -724,6 +724,15 @@ func (c *ChannelArbitrator) stateStep(triggerHeight uint32,
 		}
 		closeTx = closeSummary.CloseTx
 
+		// Before publishing the closing tx, we persist it to the
+		// database, such that it can be republished if something goes
+		// wrong.
+		if err := c.cfg.MarkCommitmentBroadcasted(closeTx); err != nil {
+			log.Errorf("ChannelArbitrator(%v): unable to "+
+				"mark commitment broadcasted: %v",
+				c.cfg.ChanPoint, err)
```

Comment:
feels like the error be returned here?

Reply:
Done: https://github.com/lightningnetwork/lnd/pull/3016/commits/37253551e8b6278b33863a7d50566182f21701e7